lib/spack/spack/environment.py

```diff
@@ -514,6 +516,7 @@ def __init__(self, path, init_file=None, with_view=None):
                 path to the view.
         """
         self.path = os.path.abspath(path)
+        self.concretization = False
```

Comment:
Sorry @alalazo one more thing: this should probably be `None` (or at least I don't think `False` is meaningful anymore) potentially with a comment about how this must be set later.

Reply:
Done